lib/upnp/igd_service.go

```diff
@@ -97,10 +203,10 @@ func (s *IGDService) DeletePortMapping(ctx context.Context, protocol nat.Protoco
 // GetExternalIPAddress queries the IGD service for its external IP address.
```

Comment:
Docs should match function name.
```suggestion
// GetExternalIPv4Address queries the IGD service for its external IP address.
```